sstables/sstables.cc

```diff
@@ -1971,7 +1971,7 @@ void maybe_add_summary_entry(summary& s, const dht::token& token, bytes_view key
         state.next_data_offset_to_write_summary += state.summary_byte_cost * entry_size;
```

Comment:
should be "sstables" instead of "stables" in the commit log

Reply:
Fixed.